client_2x/src/store/index.ts

```diff
@@ -0,0 +1,11 @@
+import Vue from "vue";
+import Vuex from "vuex";
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
```

Comment:
Nice, we should use the store asap, stuff like stored filters (printer group dropdown) and server cache data.

Reply:
Oh plan too! Printer Information, Settings, Permissions was what I thought off top of my head.

Reply:
Oh and notifications/alerts